sm/engine/queue.py

```diff
@@ -347,32 +347,37 @@ def close_connection(self):
         self.logger.info('Closing connection')
         self._connection.close()
 
-
+# TODO: Currently, a new connection with the queue is created for every queue action.
+# Though it's fine for the actual SM use-cases, where messages are published not frequently,
+# it can be a bottleneck in the future. To overcome this, the code below and annotation job
+# should be rewritten in asyncronous fashion
```

Comment:
@iprotsyuk what was the motivation for creating a new connection for every message to publish?